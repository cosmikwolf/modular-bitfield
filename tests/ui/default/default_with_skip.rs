use modular_bitfield::prelude::*;

#[bitfield]
pub struct DefaultWithSkip {
    #[default = true]
    #[skip]
    pub field: bool,
    pub other: B7,
}

fn main() {}