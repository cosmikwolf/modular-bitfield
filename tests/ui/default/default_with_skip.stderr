error: cannot use #[default(...)] on field that skips setters
 --> tests/ui/default/default_with_skip.rs:5:7
  |
5 |     #[default(true)]
  |       ^^^^^^^^^^^^^

error: field skips setters here
 --> tests/ui/default/default_with_skip.rs:6:7
  |
6 |     #[skip]
  |       ^^^^
